src/backend/joanie/core/api/admin.py

```diff
@@ -2,68 +2,102 @@
 Admin API Endpoints
 """
 import django_filters.rest_framework
-from rest_framework import authentication, permissions, viewsets
+from rest_framework import authentication, mixins, permissions, viewsets
 
 from joanie.core import filters, models, serializers
 
 
 # pylint: disable=too-many-ancestors
-class AdminViewSet(viewsets.ModelViewSet):
-    """Admin ViewSet used as base by all admin ViewSets to restrict to admin users only."""
-
-    authentication_classes = [authentication.SessionAuthentication]
-    permission_classes = [permissions.IsAdminUser, permissions.DjangoModelPermissions]
+class HasAdminPermissions(permissions.IsAdminUser, permissions.DjangoModelPermissions):
+    """
+    Default permissions for admin endpoints. It ensures that the user is an admin and
+    has the required permissions for the model.
+    """
 
 
-class OrganizationViewSet(AdminViewSet):
+class OrganizationViewSet(viewsets.ModelViewSet):
     """
     Admin Organization ViewSet
     """
 
+    authentication_classes = [authentication.SessionAuthentication]
+    permission_classes = [HasAdminPermissions]
```

Comment:
Permissons can be composed, so this should work:
```suggestion
    permission_classes = [permissions.IsAdminUser & permissions.DjangoModelPermissions]
```

Note: it supports & (and), | (or) and ~ (not).

Reply:
The main idea here was to keep admin permission factorized

Reply:
but actually it is less clear and makes you wonder why IsAdmin and HasAdmin! These are 2 orthogonal checks and it has value to keep them distinct?